packages/data-stores/test/contextual-help.tsx

```diff
@@ -31,48 +31,46 @@ async function expectNoRedirect( link: string ) {
 	} );
 }
 
-describe.skip( "disable tests because they're too flaly", () => {
```

Comment:
We don't want to enable the test suite, it's too slow and it delays everyone's PRs. I just gave the instructions to help you run the tests. 

We can revert this file as it was.